src/styles/index.scss

```diff
@@ -1,3 +1,247 @@
-body {
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white-accent: #fff;
+$card-title-color: #34568b;
+
+.header {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-accent;
   margin: 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  box-sizing: border-box;
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+    display: flex;
+    align-items: center;
+    margin-top: 2px;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__links {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  &__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    line-height: 60px;
+  }
+
+  &__item {
+    display: inline-block;
+    margin-left: 17px;
+    text-align: center;
+    height: 60px;
+    line-height: 60px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    cursor: pointer;
+    display: flex;
+    color: black;
+  }
+
+  &__link:hover {
+    color: $blue-accent;
+  }
+
+  .is-active {
+    color: $blue-accent;
+    position: relative;
+  }
+
+  .is-active::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    background-color: $blue-accent;
+    border-radius: 2px;
+  }
+}
+
+.product-card {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-accent;
+  margin: 0;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: repeat(1, 408px);
+  gap: 46px 48px;
+
+  grid-auto-flow: row;
+  place-content: center;
+
+  padding: 50px 40px;
+}
```

Comment:
Consider using a mixin for the font properties as they are repeated in both `.header` and `.product-card`. This will help you to adhere to the DRY principle and make it easier to manage font styles.